src/modules/categories/components/category/category.jsx

```diff
@@ -41,7 +41,7 @@ export default class Category extends Component {
     const s = this.state
 
     const isNullCategory = p.category === 'null-category' && p.popularity === 0
```

Comment:
`p.popularity` is required to be a string, so this comparison should be `&& p.popularity === '0'`.

Reply:
existing code, will rebase to see if fixed as part of bignumber changes.